client/signup/steps/store-features/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import intentImageUrl from 'calypso/assets/images/onboarding/intent.svg';
+import paymentBlocksImage from 'calypso/assets/images/onboarding/payment-blocks.svg';
+import wooImage from 'calypso/assets/images/onboarding/woo.png';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { saveSignupStep } from 'calypso/state/signup/progress/actions';
+import { shoppingBag, truck } from '../../icons';
+import SelectItems, { SelectItem } from '../../select-items';
+import './index.scss';
+
+interface Props {
+	goToNextStep: () => void;
+	isReskinned: boolean;
+	signupDependencies: any;
+	stepName: string;
+	initialContext: any;
+}
+
+export default function StoreFeaturesStep( props: Props ): React.ReactNode {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const headerText = translate( 'Setup your store' );
+	const subHeaderText = translate( "Let's create a website that suits your needs" );
+
+	const { stepName } = props;
+
+	// Only do following things when mounted
+	React.useEffect( () => {
+		dispatch( saveSignupStep( { stepName } ) );
+	}, [] );
+
+	const intents: SelectItem< any >[] = [
+		{
+			key: 'simple',
+			title: translate( 'Start Simple' ),
+			description: (
+				<>
+					<span className="store-features__requirements">{ translate( 'Free' ) }</span>
```

Comment:
Spans for everything ends up being semantically meaningless; instead, default to `p`s when they're a block or line of their own. I could see markdown like this (old-school `div`s are options too):

<img width="666" alt="Screen Shot 2022-01-25 at 2 42 31 PM" src="https://user-images.githubusercontent.com/349751/151072450-31685568-9760-4855-b45a-7f77bc246d66.png">

Reply:
I totally agree :) 
In fact, I tried to do it "right" the first time, but the whole description is a `p` in the wrapper component:

![image](https://user-images.githubusercontent.com/3801502/151164019-54dcf5a7-80ac-49e8-bc94-efecc36f015b.png)


So I went with the `span`s for now. I'll look at the wrapper component to see what can be done without breaking other views.

Reply:
One idea to keep the markup meaningful is to replace the `<p>` from `SelectItems`'s markup for a generic `<div>` .

`div`s allow us to have nested `<p>` and other tags as well.

We could then add a `<p>` to the existing child components - keeping their meaning - and only 4 files would be affected by this change.